.github/workflows/unittest.yml

```
@@ -1 +1,177 @@
+    
+    # 機密ファイルのクリーンアップ
+    - name: Cleanup Secret Files
+      if: always()
+      run: |
+        rm -f DotenkoV3/Config/Config.swift DotenkoV3/Info.plist DotenkoV3/GoogleService-Info.plist
+        echo "🧹 Secret files cleaned up"
+    
+    # テスト結果のアーティファクト保存
+    - name: Upload Test Results
+      if: always()
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)